packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -36,14 +36,6 @@ async function clientConfig(env) {
 		polyfills: resolve(__dirname, './polyfills'),
 	};
 
-	if (!isProd) {
-		entry.bundle = [
```

Comment:
Removed the unneeded devServers

Reply:
Does this automatically get added by the plugin / config option?

Reply:
This is part of the `DevServer.hot` option